bin/cli.class.php

```diff
@@ -430,6 +430,14 @@ public function run() {
         continue;
       }
       $this->row++;
+      if ($this->row % 1000 == 0) {
+        // Reset PEAR_DB_DATAOBJECT cache to prevent memory leak
+        $GLOBALS['_DB_DATAOBJECT']['RESULTS'] = array();
```

Comment:
Are there any side effects of clearing the results cache? If not then would it be an option to disable it completely? Presumably this would also resolve the large import problem that this ticket addresses.